File: PHP/Account.php
<?php
class Account {
    public $id;
    public $name;
    public $document;
    public $email;
    public $password;
    //creamos nuestro método constructor
    public function __CONSTRUCT($name, $document) {
        $this->name = $name;
        $this->document = $document;
    }
}
//Creamoas nuestra clase con los atributos indicados
?>

File: Python/account.py
class Account:
    id         = int
    name       = str
    document   = str
    email      = str
    password   = str
    
# Creamos nuestra clase en la cual definimos datos tipo int y str(string)

    def __init__(self, name, document):
        self.name       = name
        self.document   = document 

#creamos nuestro método constructor